Automattic/wp-calypso
# FSE: Add additional image block keywords

#### Changes proposed in this Pull Request

Add additional keywords to the image block so that it shows up for the following searches:

* Logo
* Brand
* Emblem
* Hallmark

<img width="444" alt="Screen Shot 2019-08-01 at 8 08 49 PM" src="https://user-images.githubusercontent.com/1464705/62341434-32b0ff80-b498-11e9-8d07-ea1e1171406b.png">


#### Testing instructions

* Try a search for any of the above keywords, the image block should show up in the picker.
